package-lock.json

```diff
@@ -86,9 +86,9 @@
       }
     },
     "@types/node": {
-      "version": "8.10.9",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-8.10.9.tgz",
-      "integrity": "sha512-GUUTbeDaJSRaoLkqVQ5jwwKbDiLWFX3JrKLvC078q2P51Z9Dcb5F5UdnApSYqdMk4X0VrKod1gzeoX8bGl8DMg=="
+      "version": "8.10.29",
```

Comment:
we use node 8.10 here and 8.11 in the flathub version. Intentional?

Reply:
Hmm not sure about this. This is an indirect dependency of Electron2 for the types definitions of TypeScript. I don't see any issue running the app. But yeah maybe is good to fix the Node version in the app to the one desired by Electron.

```
    "electron": {
      "version": "2.0.8",
      "resolved": "https://registry.npmjs.org/electron/-/electron-2.0.8.tgz",
      "integrity": "sha512-pbeGFbwijb5V3Xy/KMcwIp59eA9igg2br+7EHbbwQoa3HRDF5JjTrciX7OiscCA52+ze2n4q38S4lXPqRitgIA==",
      "requires": {
        "@types/node": "8.10.29",
        "electron-download": "3.3.0",
        "extract-zip": "1.6.7"
      }
```

Reply:
Don't think it's a blocker. If it works in local testing, merge away.